test/cpp_extensions/xpu_extension.sycl

```diff
@@ -0,0 +1,65 @@
+// NOTE: this is a cuda_extension.cu kernel converted to SYCL
```

Comment:
Do we still need this comments? @dvrogozh

Reply:
I left this comment since I thought it might be useful (in a sense of support) to highlight that this sycl kernel is basically the same as the noted cuda kernel. If you think that's not needed, I can drop.

Reply:
better to remove it, not sure it gives right message to user.

Reply:
removed